src/components/TooltipLearnMoreModal/index.tsx

```diff
@@ -11,6 +11,7 @@ const useStyles = makeStyles((theme: Theme) => ({
     marginRight: theme.spacing(2),
   },
   content: {
+    color: theme.palette.ClrBaseGray800,
```

Comment:
Text color wasn't set previously, which resulted in text content appearing to be missing in the tooltips learn more modal when viewing viability tests.